system/services/assetManager/AssetManagerService.cfc

```diff
@@ -936,6 +936,29 @@ component {
 		return settings;
 	}
 
+	public any function listEditorDerivates(){
+		var derivatives = _getConfiguredDerivatives();
+		var publicDerivatives = structNew();
+		var value = [];
+		var dimension = [];
+
+		for( var derivative in derivatives ) {
+
```

Comment:
The conditional logic below can be expressed more concisely:

    if ( IsBoolean( derivatives[ derivative ].inEditor ?: "" ) && derivatives[ derivative ].inEditor ) {
        // ...
    }